src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -163,7 +163,7 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
                 <Monaco
                   language='json'
                   body={data.template}
-                  height={'800px'}
+                  height={'35vh'}
```

Comment:
Is this intentional?
This could result in some strange heights on this control on smaller screens

Reply:
Yeah.. but I've realized the height issue was broken because of a recent PR that adjusted the response area heights...
So on localhost, all the different Pivots (i.e Adaptive Cards, Code Snippets etc) had different heights and the page was behaving terribly...
I'll actually leave this and as it was and create a separate issue to track the height problem